experiments/azure/cyclecloud/cpu/experiment/configs/lammps/slurm-lammps-32n.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+#SBATCH --job-name=lammps-32n
+#SBATCH --nodes=32
+#SBATCH --time=0:10:00
```

Comment:
We are setting the max time to 10 minutes across the board? I think this is OK for a lot, but I know laghos has gone over and a few others might.